### ISSUE TITLE
[FEATURE] Add yorik phase

### ISSUE THREAD

**Issue by kawaiinekololi** [CCBlueX/LiquidBounce-Issues#3945]
A phase mode by yorik100 which should bypass CubeCraft and maybe work on Hypixel in corners.

**Comment by hahayesfun** [comment]
Good idea, maybe make more CC bypasses in the future (it's a nice server to cheat on, but they updated their anticheat a lot)
